deltacat/compute/converter/utils/convert_task_options.py

```diff
@@ -0,0 +1,90 @@
+from typing import Optional, Dict
+from deltacat.exceptions import RetryableError
```

Comment:
FYI, I have assumed you've done the due diligence to ensure the estimation is accurate. Not going into greater depth on this business logic.